documentation/asciidoc/computers/remote-access/ssh.adoc

```diff
@@ -0,0 +1,176 @@
+[[ssh]]
+== Access a remote terminal with SSH
+
+You can access the terminal of a Raspberry Pi remotely from another computer on the same network using the **S**ecure **SH**ell (SSH) protocol.
+
+=== Enable the SSH server
+
+By default, Raspberry Pi OS disables the SSH server. Enable SSH in one of the following ways:
```

Comment:
Could also mention that you can enable it during image-flashing using RPi Imager?